answers/Go/@codehakase/01-name-concatenation.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func main() {
+	var (
+		firstname string
+		lastname  string
+		age       int
+	)
+	fmt.Print("Enter first name: ")
+	fmt.Scanf("%s", &firstname)
```

Comment:
I suggest replacing `fmt.Scanf` in lines 12, and 15 with `fmt.Scanln` e.g.

```go
fmt.Scanln(&firstname)
```

With `Scanf`, it tends to skip the second input, and go to the third. See:

![why you should replace scanf with scanln](https://user-images.githubusercontent.com/11996508/60949488-3a441500-a2ed-11e9-9d35-b841d7c45ec4.png)

Reply:
Another option would be to place `\n` after the interpolated input string, when using `Scanf`, e.g.

```go
fmt.Scanf("%s\n", &firstname)
```